dark/reads.py

```diff
@@ -1512,6 +1513,37 @@ def variableSites(self, confirm=False, homogeneityLevel=1.0,
 
         return varSites
 
+    def combineReads(self):
+        """
+        Combine all reads into a single read.
+
+        @return: a C{str} sequence made from combining all reads.
+        """
+        reads = list(self)
+        assert len({len(read) for read in self.reads}) == 1
+
+        sequence = ''
+        for bases in zip_longest(*(read.sequence for read in reads),
+                                 fillvalue='-'):
+            bases = set(bases)
+            if len(bases) == 1:
+                sequence += bases.pop()
+            elif (len(bases) == 2 and 'N' in bases and bool(
```

Comment:
The `bool` isn't needed here (but doesn't hurt) since an empty set is considered `False` (like all empty things in Python).